tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_device_info.py

```diff
@@ -13,10 +13,16 @@ def test_that_device_info_is_populated(self):
         settings.launch()
         device_info = settings.open_device_info_settings()
 
+        # devices without sim cards have no phone number
+        if 'local_phone_numbers' in self.testvars:
+            self.assertTrue(len(device_info.phone_number) > 0)
+            self.assertIsNotNone(device_info.phone_number)
+        else:
+            self.assertEqual(device_info.phone_number, '')
```

Comment:
Right, I was forgetting about the template. I'll change this to use `if self.testvars.get('local_phone_numbers')` as this will return None if the key is not present, or an empty string/list, all of which are falsey. I'll edit the template to have an empty list for now - once the remote lab changes are in place the value will be either a string or list, and shouldn't be a list with only one item. We can then handle that in the tests.